app.js

```diff
@@ -1,156 +1,182 @@
 "use strict";
 
-let userName = prompt("What is your name?");
-let points = 0;
-{
-  userName = userName.toUpperCase();
-  alert("Hello and welcome "+ userName + "!");
-  points++;
-  console.log(points);
-}
+function questions(answer) {
```

Comment:
This function seems to limit you to only yes answers, one way around that would be to add a second argument, being yes or no, to check against.